timemachine/md/minimizer.py

```diff
@@ -170,55 +196,131 @@ def minimize_host_4d(
         raise ValueError("mols must be length 1 or 2")
 
     mass_list = [np.array(host_masses)]
-    conf_list = [np.array(host_config.conf)]
+    conf_list = [minimized_host_coords]
     for mol in mols:
-        # mass increase is to keep the ligand fixed
-        mass_list.append(get_mol_masses(mol) * 100000)
+        # Set ligand masses to inf to ensure ligands don't move
+        mass_list.append(np.ones(mol.GetNumAtoms()) * np.inf)
 
-    if mol_coords is not None:
-        for mc in mol_coords:
-            conf_list.append(mc)
-    else:
-        for mol in mols:
-            conf_list.append(get_romol_conf(mol))
+    if mol_coords is None:
+        mol_coords = [get_romol_conf(mol) for mol in mols]
+    for mc in mol_coords:
+        conf_list.append(mc)
 
     combined_masses = np.concatenate(mass_list)
     combined_coords = np.concatenate(conf_list)
 
     hgt = topology.HostGuestTopology(host_bps, top, host_config.num_water_atoms)
 
-    # this value doesn't matter since we will turn off the noise.
-    seed = 0
+    dt = 1.5e-3
+    friction = 1.0
 
-    intg = LangevinIntegrator(0.0, 1.5e-3, 1.0, combined_masses, seed).impl()
+    intg = LangevinIntegrator(DEFAULT_TEMP, dt, friction, combined_masses, seed).impl()
```

Comment:
nit: would it be worth accepting a `temperature` argument here, versus assuming `DEFAULT_TEMP`?

Reply:
Lol it accepts the temperature parameter and I just don't use it. Will fix it.

Reply:
Fixed in https://github.com/proteneer/timemachine/pull/1339/commits/130b5e2c1ae9033dc9494a224b4888feced19801